how_to_use.md

```diff
@@ -4,51 +4,59 @@ title: How to use Data Commons
 nav_order: 1
 ---
 
-{: .no_toc}
 # How to use Data Commons
 
-* TOC
-{:toc}
+Data Commons offers many different ways to consume its data: 
 
-## Learn about the data in Data Commons
+- [Interact with the website](#interact), datacommons.org
+- [Learn about the data sources](#learn)
+- [Query the base data directly](#query), interactively and programmatically
+- [Embed data visualizations](#embed) in your own site
+- [Download data in CSV format](#download), for offline analysis
 
-To find out what data is available in Data Commons, check out the [Statistical Variable Explorer](https://datacommons.org/tools/statvar) and see the [Data sources](/datasets/index.html) pages.
+There are also several options for providing new data to Data Commons:
+- [Contribute data to the base Data Commons](#contribute-data)
+- [Host your own data site](#custom) using Data Commons' web framework 
 
-## Issue interactive data queries
+## Interact with the datacommons.org website {#interact}
 
-For quick analysis, use the search query bar on the [home page](https://datacommons.org/) or use any of the visualization tools, such as the [Timeline](https://datacommons.org/tools/visualization#visType=timeline), [Scatter](https://datacommons.org/tools/visualization#visType%3Dscatter), and [Map](https://datacommons.org/tools/visualization#visType%3Dmap) explorers.
+For quick analysis, use the search query bar on the [home page](https://datacommons.org/) or use any of the visualization tools, such as the [Timeline](https://datacommons.org/tools/visualization#visType=timeline), [Scatter](https://datacommons.org/togitols/visualization#visType%3Dscatter), and [Map](https://datacommons.org/tools/visualization#visType%3Dmap) explorers.
 
-If you want to issue SQL queries, and you have a Google Cloud Platform account, use BigQuery Studio on Data Commons data in [Analytics Hub](https://cloud.google.com/analytics-hub). See the [Data Commons in BigQuery](/bigquery/index.html) page for more details.
+## Learn about the data in Data Commons {#learn}
 
-## Issue programmatic data queries
+To find out what data is available in Data Commons, check out the [Statistical Variable Explorer](https://datacommons.org/tools/statvar) and see the [Data sources](/datasets) pages.
```

Comment:
Wonder if Statistical Variable might be a foreign concept and if its worth linking to (or having a sentence with) its definition?

Reply:
Let's skip that since there's a dedicated doc about it now.